libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -69,4 +70,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://docs.gupshup.io/docs/send-single-message',
     logoFileName: { light: 'gupshup.png', dark: 'gupshup.png' },
   },
+  {
+    id: SmsProviderIdEnum.Whatsapp,
+    displayName: 'Whatsapp',
+    channel: ChannelTypeEnum.SMS,
+    credentials: whatsappConfig,
+    docReference: 'https://developers.facebook.com/docs/whatsapp/',
+    logoFileName: { light: 'whatsapp.svg', dark: 'whatsapp.svg' },
+  },
```

Comment:
@ainouzgali I would love to have the whatsapp integration with the "Beta" ribbon you created too, do you remember the property we need to add here for it to work?

Reply:
I see I should add `betaVersion: true`